src/Writer/XlsWriter.php

```diff
@@ -78,8 +78,11 @@ public function write(array $data): void
         $this->init($data);
 
         fwrite($this->file, '<tr>');
+        // prevent xls injection
+        $patterns = ['/^=/', '/^\+/', '/^-/', '/^@/'];
+        $replace = ['!=', '!+', '!-', '!@'];
         foreach ($data as $value) {
-            fwrite($this->file, sprintf('<td>%s</td>', $value));
+            fwrite($this->file, sprintf('<td>%s</td>', preg_replace($patterns, $replace, $value)));
```

Comment:
Please apply the same here.

Reply:
Yes good point - no need of those variables being set.